clappr/clappr-ios
# Media control with panels

# Goal

Creating new media control with panels and separating in plugins his components

# How to test

- Run all tests
- Make sure that all media control functionalities are functional
- Make sure that tvOS is functional
- Test with different scenarios, like entering/exiting fullscreen, rotate, seek, etc
